test/maps/make_simple_shp.py

```diff
@@ -0,0 +1,30 @@
+import shapefile

```

Comment:
The return of the Python testing scripts! 😉 (I don't love that we have to do this—it says something about the maturity of the Julia geospatial ecosystem—but it's fine by me for now.)